pkg/skaffold/docker/parse.go

```diff
@@ -206,6 +206,11 @@ func extractCopyCommands(nodes []*parser.Node, onlyLastImage bool, cfg Config) (
 		switch node.Value {
 		case command.From:
 			from := fromInstruction(node)
+			if from.image == "" {
```

Comment:
out of curiosity, I tried this out with an empty `FROM` instruction in a Dockerfile. this is broken but I wanted to see what skaffold would do....https://github.com/GoogleContainerTools/skaffold/issues/4935


```
➜  microservices git:(master) skaffold build
Generating tags...
 - leeroy-web -> gcr.io/nkubala-demo/leeroy-web:v1.15.0-22-ga549c7a1d
 - leeroy-app -> gcr.io/nkubala-demo/leeroy-app:v1.15.0-22-ga549c7a1d-dirty
Checking cache...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x4bbea4a]

goroutine 39 [running]:
github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker.fromInstruction(0xc000926850, 0xc0009e7660, 0x5a57961, 0x7, 0xc0009e7718)
        /Users/nkubala/Development/skaffold/pkg/skaffold/docker/parse.go:361 +0x2a
github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker.expandOnbuildInstructions(0xc00093c940, 0x8, 0x8, 0x5d99020, 0xc000540840, 0x0, 0x0, 0xc0009e7878, 0x40b069b, 0xc0009300d0)
```

not broken by your PR, but we should consider what this interaction looks like for broken Dockerfiles. after this other bug gets fixed, will your code handle the broken Dockerfile situation gracefully?

Reply:
Put in a fix for #4935. Tested with this branch and it behaves correctly IMO.

Reply:
👍